src/views/DrawerNavigatorItems.tsx

```diff
@@ -54,13 +67,24 @@ const DrawerNavigatorItems = ({
       const scene = { route, index, focused, tintColor: color };
       const icon = renderIcon(scene);
       const label = getLabel(scene);
-      const accessibilityLabel = typeof label === 'string' ? label : undefined;
+      const accessibleLabel =
+        typeof accessibilityLabel === 'string'
```

Comment:
I'd change this to

```suggestion
        accessibilityLabel !== undefined
```

If user passes an incorrect type, it should error instead of silently ignoring it